### ISSUE TITLE
CFE logs indicate "found Elastic IPs: undefined" although failover appears to work correctly

### ISSUE THREAD

**Issue by mikeoleary** [F5Networks/f5-cloud-failover-extension#145]
## Do you already have an issue opened with F5 support?
No. Customer is working with me (F5 SA).

## Description
F5-provided CFT was deployed to create HA pair in same AZ. CFE was installed by the template. Customer is failing over IP addresses only (no route table updates).

It appears that failover is working correctly. About 8 private IP addresses, each with associated EIP's, are correctly moved between 2 interfaces in AWS. However, CFE logs include this line: 

>Thu, 01 Aug 2024 15:23:17 GMT - fine: [f5-cloud-failover] _discoverAddressOperations found Elastic IPs: undefined

Question for PM:
- what could cause this, and how can we fully investigate further? I can provide CFE logs directly if you give me an email address, or over MS Teams.

Other details:
- IAM permissions were configured by CFT and RoleAssignment correctly links IAM Role to EC2 instances.
- default routes allow communication outbound to Internet

## Environment information
For bugs, enter the following information:
- Cloud Failover Extension Version: 1.2
- BIG-IP version: 17.1.1
- Cloud provider: AWS

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 4

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by mikeshimkus** [comment]
@mikeoleary Did you mean v2.1.2? We just fixed an issue where we saw errors when failoverAddresses was disabled, and we have another fix for the same behavior with failoverRoutes that will be released in the next couple of week. If you can send the logs I will have a look to see if it's a new issue.

**Comment by mikeoleary** [comment]
@mikeshimkus , yes, I meant the latest version of CFE. In our case, we have not defined failoverRoutes at all. I'll send you a message over Teams with the logs, which will also contain the CFE config, and I will share the IAM Role definition with you too.

**Comment by mikeshimkus** [comment]
Looks very similar, I created issue EC-541 to investigate and will fix if not the same.
